docs/pages/admin-guides/access-controls/okta/app-and-group-sync.mdx

```diff
@@ -0,0 +1,209 @@
+synchronizer creates the following resources for each matched Okta group or
+application:
+
+- A role for members that grants access to the group/application.
+- A role for owners that grants the ability to review access to the group/application.
+- An Access List representing membership to the group/application.
+- Members for the Access List.
+
+It should be noted that the Access List sync waits until the Okta groups and Okta applications
+has finished syncing as Teleport resources, so it may not start synchronizing immediately on startup.
+
+Only Okta user groups with assignments will be imported as Teleport Access
+Lists. If an Okta User Group has no assignments, it will not be imported until
+it has assignments. If the last user is removed from the Access List, the Access
+List will be removed from Teleport on the next sync.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- Teleport Identity enabled on your account.
+
+- An Okta authentication connector. 
+
+  <Admonition>
+  Before following the guided app and group sync integration flow, you must have
+  completed the following:
+  - [Guided Okta single sign-on flow](./guided-sso.mdx).
+  - [Guided Okta user sync flow](./user-sync.mdx).
+  </Admonition>
+
+- (Optional) The Okta SCIM integration. You can follow the [guided integration
+  flow](./scim-integration.mdx).
+
+- (!docs/pages/includes/tctl.mdx!)
+
+- An Okta organization with admin access.
+
+Enabling the Okta integration will make Teleport take ownership over app and
+group assignments in Okta and can make changes within Okta based on your
+Teleport RBAC configuration. To limit the scope of the integration, ensure that:
+
+- (Optional) You have organized your Okta applications and groups into [Okta
+  resource
+  sets](https://help.okta.com/en-us/content/topics/security/custom-admin-role/create-resource-set.htm),
+  which allow you to limit the scope your Okta access token.
+
+## Step 1/2. Set default list owner(s)
+
+The first step of setting up Okta Access List synchronization is defining
+default Access List owners. You can select any number of default owners or
+manually enter the owners if they do not yet exist in the system.  These owners
+can be later changed and will not be overwritten by the Okta Access List
+synchronization process.
+
+1. Make sure you are on the **Sync User Groups and App Assignments** page. If
+   you are not, visit the integration status page by navigating to **Access** ->
+   **Integrations** and clicking anywhere on the Okta integration row. Then
+   visit the **Sync User Groups and App Assignments** page.
+1. Under **Add Default List Owner(s)**, enter the names of Teleport users to the
+   dropdown menu.
+
+## Step 2/2. Configure user group and application imports
+
+After configuring default owners, you can customize which Okta user groups and
+applications should be imported. By default, all user groups and applications
```

Comment:
The filters are a bit unintuitive UX-wise... If "sync all apps/groups" are unchecked but no filters are provided, this is treated the same as providing a wildcard `*`. Worth mentioning, imo